x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/form_validation/hot_phase_validation.test.ts

```diff
@@ -50,94 +51,131 @@ describe('<EditPolicy /> hot phase validation', () => {
       expect(actions.rollover.hasSettingRequiredCallout()).toBeTruthy();
     });
 
-    test(`doesn't allow -1 for max primary shard size`, async () => {
-      await actions.rollover.toggleDefault();
-      await actions.rollover.setMaxPrimaryShardSize('-1');
+    describe('max primary shard size', () => {
+      test(`doesn't allow -1`, async () => {
+        await actions.rollover.toggleDefault();
+        await actions.rollover.setMaxPrimaryShardSize('-1');
 
-      actions.errors.waitForValidation();
+        actions.errors.waitForValidation();
 
-      actions.errors.expectMessages([i18nTexts.editPolicy.errors.numberGreatThan0Required]);
-    });
+        actions.errors.expectMessages([i18nTexts.editPolicy.errors.numberGreatThan0Required]);
+      });
 
-    test(`doesn't allow 0 for max primary shard size`, async () => {
-      await actions.rollover.toggleDefault();
-      await actions.rollover.setMaxPrimaryShardSize('0');
+      test(`doesn't allow 0`, async () => {
+        await actions.rollover.toggleDefault();
+        await actions.rollover.setMaxPrimaryShardSize('0');
 
-      actions.errors.waitForValidation();
+        actions.errors.waitForValidation();
 
-      actions.errors.expectMessages([i18nTexts.editPolicy.errors.numberGreatThan0Required]);
+        actions.errors.expectMessages([i18nTexts.editPolicy.errors.numberGreatThan0Required]);
+      });
     });
 
-    test(`doesn't allow -1 for max size`, async () => {
-      await actions.rollover.toggleDefault();
-      await actions.rollover.setMaxSize('-1');
+    describe('max primary docs size', () => {
+      test(`doesn't allow -1`, async () => {
+        await actions.rollover.toggleDefault();
+        await actions.rollover.setMaxPrimaryDocsSize('-1');
```

Comment:
I would suggest naming action `setMaxPrimaryShardDocs` to follow the logic of `maxSize`/`maxDocs` and `maxPrimaryShardSize`/`maxPrimaryShardDocs`